src/components/data-management/UploadDetailsModal.jsx

```diff
@@ -39,7 +36,7 @@ const UploadDetailsModal = (props) => {
 
   useEffect(() => {
     if (replacementFileBundle) {
-      bundleToFile(replacementFileBundle, SELECTED_TECH).then((newFile) => {
+      fileObjectToFileRecord(replacementFileBundle, SELECTED_TECH).then((newFile) => {
```

Comment:
You have the "File bundle" syntax you are changing in other places too, like `uploadFileBundle`

Reply:
If part of what you are doing in this ticket is renaming file bundle to file object I would change it everywhere

Reply:
replaced